app/src/ingestion/pdf_postprocess.py

```diff
@@ -151,6 +163,16 @@ def _group_list_texts(markdown_texts: list[EnrichedText]) -> list[EnrichedText]:
     for current_text in markdown_texts[1:]:
         previous_text = grouped_texts[-1]
 
+        # Unstructured text sometimes splits a bullet from it's text;
```

Comment:
```suggestion
        # Unstructured text sometimes splits a bullet from its text;
```

Reply:
However I think this is already being done earlier by the code after `# Sometimes Unstructured splits a ListItem into an empty ListItem` above, so we may be able to remove this code

Reply:
Yea, there's some overlap and it could be cleaned up.